code/controllers/subsystem/vote.dm

```diff
@@ -239,6 +239,10 @@ SUBSYSTEM_DEF(vote)
 				question = "Map for next round"
 				for(var/x in subtypesof(/datum/map))
```

Comment:
couldn't this just be 
```dm
var/datum/map/M in subtypesof(/datum/map)

Reply:
I think so? I did it like this before I was using initial() for stuff so it might work.

Reply:
Shouldn't because `subtypesof`/`typesof` returns a list of types, while `var/datum/map/M` is searching for an instance.